src/config/index.ts

```diff
@@ -8,7 +8,7 @@ import {
 const configSchema = {
   DEPLOYMENT_LOGS: parseBoolean({ default: null }),
   FORK_MAINNET: parseBoolean({ default: false }),
-  FORK_BLOCK_NUMBER: parseInteger({ default: 13592927 }),
+  FORK_BLOCK_NUMBER: parseInteger({ default: 13678600 }),
```

Comment:
Block number of stark proxy fix execution: https://etherscan.io/tx/0xcaba0c11db92c800a6f675415abe3d1e98d06473bed2e4570f5b2ea179a65c5d